src/engine/controls/bpmcontrol.cpp

```diff
@@ -93,6 +93,61 @@ BpmControl::BpmControl(const QString& group,
             this,
             &BpmControl::slotTranslateBeatsMove,
             Qt::DirectConnection);
+    
+    m_pBeatsHalve = new ControlPushButton(ConfigKey(group, "beats_set_halve"), false);
```

Comment:
We try to avoid plain new because it is easy to forget the corresponding delete. 
Please use `std::make_unique`

Reply:
Apart from that this PR works as desired :+1:

Reply:
What would the syntax be? Not `    m_pBeatsHalve = std::make_unique<ControlPushButton>(ControlPushButton(ConfigKey(group, "beats_set_halve"), false));
` or `    m_pBeatsHalve = std::make_unique<ControlPushButton>((ConfigKey(group, "beats_set_halve"), false));`..

Reply:
The latter, exactly as I wrote. just without the double ((
Actually, the boolean `bPersists` can also be omitted since it defaults to `false`.
And we switch to use QStringLiteral()
So it would be
`m_pBeatsHalve = std::make_unique<ControlPushButton>(ConfigKey(group, QStringLiteral("beats_set_halve")));` etc.

Reply:
Ah thanks, I don't know why the issue page didn't auto update with your comment. I'll sort this out in a bit